transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/HyperwalletTransferMethodUi.java

```diff
@@ -58,6 +59,23 @@ public static synchronized HyperwalletTransferMethodUi getInstance(
         return sInstance;
     }
 
+    /**
+     * Returns an instance that will also initialize the Insights library.
+     *
+     * @param context                     A Context of the application consuming this Intent.
+     * @param authenticationTokenProvider An implementation of the {@link HyperwalletAuthenticationTokenProvider}
+     * @return singleton instance of HyperwalletTransferMethodUi
+     */
+    public static synchronized HyperwalletTransferMethodUi getInstance(@NonNull final Context context,
```

Comment:
I think getInstance method should be like this as per section 3.2  1st diagram. Also need to remove old method getInstance or need to alternate it.
<img width="843" alt="Screenshot 2019-11-08 at 3 18 05 PM" src="https://user-images.githubusercontent.com/52773997/68466815-27116980-023b-11ea-8ef0-078dcfcd08b9.png">